contact.html

```diff
@@ -3,27 +3,129 @@
 title: Contact
 permalink: /contact.html
 coordies:
-  'Ahsan Barkati':
-    number: +917318018978
-    mail: ahsanb@iitk.ac.in
-    github: ahsanbarkati
-    image: ahsan.jpg
-  'Anant Prakash':
-    number: +917318019930
-    mail: anantp@iitk.ac.in
-    github: apwhitehat
-    image: anant.jpg
-  'Ayush Bansal':
-    number: +919462545882
-    mail: ayushb@iitk.ac.in
-    github: ayush268
-    image: ayush.jpg
-  'Parv Mor':
-    number: +917318019085
-    mail: parv@iitk.ac.in
-    github: parvmor
-    image: parv.jpg
+  'Aditya Gulati':
+    number: +910006548714
```

Comment:
This number looks weird. @pclubiitk/coordies-2019-20 ?

Reply:
Updated